src/.travis.yml

```diff
@@ -50,13 +53,15 @@ matrix:
         - android-wait-for-emulator
       before_install:
         - nvm install 6
-      script: cd src && npm run test.android
+      script:
+        - cd src && npm run test.android
     - os: osx
       language: node_js
       node_js: "6"
       jdk: oraclejdk8
       osx_image: xcode9.2
-      script: cd src && npm run test.ios
+      script:
+        - cd src && npm run test.ios
```

Comment:
The current job won't pass until we change the osx_image to 8.3 or use a command to fire up 10.x simulator for example: 
before_script:
- echo no | npm install -g ios-sim
- ios-sim start --devicetypeid "iPhone-7-Plus, 10.3"